release/models/qos/openconfig-qos-interfaces.yang

```diff
@@ -362,13 +369,13 @@ submodule openconfig-qos-interfaces {
     leaf dropped-pkts {
       type oc-yang:counter64;
       description
-        "Number of packets dropped by the queue due to overrun";
+        "Number of packets dropped by the queue due to overrun, that is tail-drop or WRED drop";
```

Comment:
I think overrun clearly is tail-drop.   But I am not sure that a queue overrun is the same as a WRED drop.  Do we need to include this clarification in this PR?  I suggest this clarification is a separate PR as I think this deserves some additional impact analysis. (ie: do 2+ platforms support this interpretation?  Do we need this clarification?)

Reply:
There is no other counter that counts WRED dropped packets. And for sure we want dropped packets to be counted.
Unless we want to define new counters, abouve is only resonable interpretation - IMHO.
If we are going to define new drop couters we should have explicit Tail-drop and WRED-drop. Perhaps at this time we should introduce support for "colored" WRED (like 3 WRED slopes per queue - follow IETF PHBs AFx.1, AFx.2, AFx.3) and corresponding drop counters.
IMHO making existing and only per-queue drop counters (`dropped-pkts`, `dropped-octets`) and aggregated couter for drop regardless of cause the best option for now.

Reply:
OC Operators comment that this description update should also mention that the queue-management profile determines if RED/WRED is configured.  For example:

"Number of packets dropped by the queue due to overrun, that is tail-drop or RED/WRED if indicated by the queue-management-profile"